src/boot/bootctl-install.c

```diff
@@ -591,7 +591,7 @@ static int efi_timestamp(EFI_TIME *ret) {
                 return log_error_errno(r, "Failed to convert timestamp to calendar time: %m");
 
         *ret = (EFI_TIME) {
-                .Year = 1900 + tm.tm_year,
+                .Year = htole16(1900 + tm.tm_year),
```

Comment:
hmm, how did this come up? EFI is LE-only, hence we so far kinda ignored the issue since HAVE_EFI is off on BE anyway.

I mean, I am not against making this stuff work on BE too, even if it could only be used "offline" but I do wonder how this came up for you, and I doubt this patch here is going to be sufficient.

Reply:
While digging through the PE format is noticed it was LE so I decided to fix these occurrences. I can drop it again since I didn't test this at all.

Reply:
yeah, maybe let's drop it, let's not get lost on that.